packer/Vagrantfile

```diff
@@ -0,0 +1,81 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "infrasim"
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = 4096
+    vb.cpus = 4
+    vb.customize ["modifyvm", :id, "--ioapic", "on", "--cpus", "4"]
+  end
+
+  config.vm.base_mac = "080027627FDA"
```

Comment:
Would suggest at least giving 2 NICs for infrasim running on virtualbox: first one for purpose of NAT connection occupied by vagrant/virtualbox, second one for control/data network of vritual node.